dffml/operation/db.py

```diff
@@ -0,0 +1,189 @@
+import inspect
+
+
+from ..base import config
+from ..db.sqlite import SqliteDatabase,SqliteDatabaseContext
+from typing import Dict,Any,Optional,List
+from ..df.base import op
+from ..db.base import Conditions
+from ..df.types import Definition
+
+
+#definitions
+QUERY_TABLE=Definition(name="query_table",primitive="str")
+QUERY_DATA=Definition(name="query_data", primitive="Dict[str, Any]")
+QUERY_CONDITIONS=Definition(name="query_conditions",primitive="Conditions")
+QUERY_COLS=Definition(name="query_cols",primitive="List[str]")
+QUERY_LOOKUPS=Definition(name="query_lookups", primitive="Dict[str, Any]")
+
+@config
+class SqliteQueryConfig:
+    """
+        query_type : "create","insert","update","remove","lookup"
+    """
+    database : SqliteDatabase
```

Comment:
Since you changed the filename, lets also change all the class, op, etc. names in this to be `db_` rather than `sqlite_` so the config would be `database: BaseDatabase`